docs/ovs-offload.md

```diff
@@ -0,0 +1,207 @@
+# OVS Hardware Offload
+
+The OVS software based solution is CPU intensive, affecting system performance
+and preventing fully utilizing available bandwidth. OVS 2.8 and above support
+new feature called OVS Hardware Offload which improves performance significantly.
+This feature allows to offload the OVS data-plane to the NIC while maintaining
+OVS control-plane unmodified. It is using SR-IOV technology with VF representor
+host net-device. The VF representor plays the same role as TAP devices
+in Para-Virtual (PV) setup. A packet sent through the VF representor on the host
+arrives to the VF, and a packet sent through the VF is received by its representor.
+
+## Supported Ethernet controllers
+
+The following manufacturers are known to work:
+
+- Mellanox ConnectX-5 and above
+
+## Prerequisites
+
+- Linux Kernel 5.7
+- Open vSwitch 2.13
```

Comment:
I think we can omit this requirement, since we will ensure that Antrea ships with the required version of the userspace OVS daemons?